<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBreakpointLabelsDialog</class>
 <widget class="QDialog" name="EditBreakpointLabelsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Breakpoint Labels</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="typeNewLabel">
       <property name="text">
        <string>T&amp;ype a new label</string>
       </property>
       <property name="buddy">
        <cstring>newLabelEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="newLabelEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="addLabelButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="chooseExistingLabel">
       <property name="text">
        <string>O&amp;r choose among existing labels</string>
       </property>
       <property name="buddy">
        <cstring>labelList</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QListWidget" name="labelList"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>newLabelEdit</tabstop>
  <tabstop>addLabelButton</tabstop>
  <tabstop>labelList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
